public/app/core/components/Page/Page.tsx

```diff
@@ -57,9 +56,7 @@ export const Page: PageType = ({
         <NativeScrollbar
           // This id is used by the image renderer to scroll through the dashboard
           divId="page-scrollbar"
-          autoHeightMin={'100%'}
```

Comment:
this prop existed on CustomScrollbars (which NativeScrollbar used as Props type) but was never actually used so removing it here